default/stringtables/en.txt

```diff
@@ -6433,7 +6433,18 @@ A set of core supply starlanes for an empire (the set of least-jump starlanes co
 
 Supply lines can also be used to refill the [[metertype METER_FUEL]] supply of ships.
 
-Supply lines will be obstructed by armed enemy ships belonging to empires with which an empire is at [[encyclopedia WAR_TITLE]]. At the opposite end of the spectrum, empires in an [[encyclopedia ALLIANCE_TITLE]] will share supply lines and all their associated benefits.'''
+Empires in an [[encyclopedia ALLIANCE_TITLE]] use each other's supply lines.
+The extension of supply lines from source planets into a system can be obstructed by armed enemy ships in that system and that are in fleets set to [[FLEET_OBSTRUCTIVE]] or [[FLEET_AGGRESSIVE]]. Enemy ships are those owned by empires that are at [[encyclopedia WAR_TITLE]] with the supply line's empire.
```

Comment:
```suggestion
The extension of supply lines from source planets into a system can be obstructed by armed enemy ships that are in that system and that are in fleets set to [[FLEET_OBSTRUCTIVE]] or [[FLEET_AGGRESSIVE]]. Enemy ships are those owned by empires that are at [[encyclopedia WAR_TITLE]] with the supply line's empire and neutral / monster ships.
```

From testing and the code, I think monsters also qualify for obstruction of supply propagation.